File: Tema2/Ejercicio010/ejercicio10.java
import java.util.Scanner;

public class ejercicio10 {

    public static void main(String[] args) {
        Scanner lector = new Scanner(System.in);
        System.out.println("Escribe tres numeros, y veremos si los dos primeros dan como solucion el tercero: ");
        int a = lector.nextInt();
        System.out.println("Dame otro numero: ");
        int b = lector.nextInt();
        System.out.println("Dame el ultimo numero: ");
        int c = lector.nextInt();

        if (a + b == c) {
            System.out.println("Si sumamos el 1er numero y el 2do el resultado es el 3ro");
        }else if (b + c == a) {
            System.out.println("Si sumamos el 2do numero y el 3ro el resultado es el 1ro");
        }else if (a + c == b){
            System.out.println("Si sumamos el 1er numero y el 3ro el resultado es el 2do");
        }else {
            System.out.println("Ninguna suma da como resultado otro numero");
        }
        lector.close();
    }

}

File: Tema2/Ejercicio010/ejercicio10.py
class ejercicio10:
    
    def suma():
        a = int(input("Introduce tres numeros y veremos si sumamos dos da como resultado el tercero: \n"))
        b = int(input("Introduce un segundo numero: \n"))
        c = int(input("Introduce un tercer numero: \n"))
        
        if a + b == c:
            print("La suma del 1er num y el 2do num tiene como resultado el tercer num")
        elif b + c == a:
            print("La suma del 2do num y el 3er num tiene como resultado el primer num")
        elif c + a == b: 
            print("La suma del 3er num y el 1er num tiene como resultado el segundo num")
        else:
            print("Ninguna de las sumas dan un numero ")
            
    if __name__ == "__main__":
        suma()